go.mod

```diff
@@ -1,3 +1,3 @@
 module github.com/invopop/gobl.xinvoice
 
-go 1.21.6
+go 1.22.0
```

Comment:
I'd suggest going with 1.19 for libraries, or at most 1.21. 1.22 is a bit too new still for most people to have caught up.